src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/OffenderProgramProfileRepository.kt

```diff
@@ -11,7 +11,7 @@ interface OffenderProgramProfileRepository : JpaRepository<OffenderProgramProfil
   fun findByCourseActivityAndOffenderBooking(
     courseActivity: CourseActivity,
     offenderBooking: OffenderBooking,
-  ): OffenderProgramProfile?
+  ): List<OffenderProgramProfile>
```

Comment:
An offender can be allocated to a course multiple times by ending then reallocating.